cleanlab/dataset.py

```diff
@@ -338,27 +317,18 @@ def overall_label_health_score(
 
     Only provide **exactly one of the above input options**, do not provide a combination.
 
-    **Parameters**: For information about the arguments to this method, see the documentation of
-    :py:func:`find_overlapping_classes <cleanlab.dataset.find_overlapping_classes>`.
-
     Examples
     --------
     >>> from cleanlab.dataset import overall_label_health_score
     >>> from sklearn.linear_model import LogisticRegression
     >>> from sklearn.model_selection import cross_val_predict
-    >>> data, labels = get_data_labels_from_dataset(yourFavoriteDataset)
+    >>> data, labels = get_data_labels_from_dataset()
     >>> yourFavoriteModel = LogisticRegression()
-    >>> pred_probs = cross_val_predict(
-            yourFavoriteModel,
-            data,
-            labels,
-            cv=3,
-            method="predict_proba",
-        )  # generate cross-validation estimates for each input data point
-    >>> score = overall_label_health_score(
-            labels=labels,
-            pred_probs=pred_probs,
-        )  # a score measuring the overall quality of all labels in a dataset.
```

Comment:
Note that docstest is failing in functions that print the output instead of returning it.

Sadly, the `# doctest: +ELLIPSIS` directive won't work in this case.

It would be nice if we could somehow suppress the printouts for doctest without changing the docstrings.

Reply:
Thanks for pointing out the reason for the failed tests. Currently, I can think of two ways to suppress the outputs. The first one is to supply the `verbose=False` parameter to the two printing functions. But this will mean two additional not-so-relevant parameters in the examples.

Another one is overriding the print function in the doctest setup code in "dataset.rst". It would look like this:
![Screen Shot 2023-04-16 at 23 46 22](https://user-images.githubusercontent.com/60683228/232373817-aa65e8ad-30be-4106-ae5b-a73031dcf47d.png)
But I am not sure why this method did not work (i.e. it seems like the built-in print function is still being used during doctest). Do you have any thoughts on the reason?

Reply:
I'm not sure if this redefines `builtin.print`. Maybe this is a namespace issue?